app/models/manageiq/providers/lenovo/provider.rb

```diff
@@ -1,9 +1,10 @@
 class ManageIQ::Providers::Lenovo::Provider < ::Provider
+  # rubocop:disable Rails/HasManyOrHasOneDependent,Rails/InverseOf
```

Comment:
Please don't do this, either add the `:dependent` and `:inverse` or just leave it and its okay if rubocop complains